source/template-apps.txt

```diff
@@ -74,20 +74,20 @@ Create a Template App
       :tabid: ui
       
       Follow the procedure described in the 
-      :ref:`Create a Realm App with Realm UI <create-a-realm-app>` guide, and select
-      the :guilabel:`Create App from Template` tab in the :guilabel:`Create a new Realm App` step. 
+      :ref:`Create an {+app+} with {+ui+} <create-a-realm-app>` guide, and select
```

Comment:
I'm guessing this actually follows the naming guidance, but holy moly, this is a horrible mouthful to read:
> Create an Atlas App Service with Atlas App Services UI guide.